Tasks/AzurePowerShellV5/azurepowershell.ts

```diff
@@ -160,6 +160,43 @@ async function run() {
     catch (err) {
         tl.setResult(tl.TaskResult.Failed, err.message || 'run() failed');
     }
+    finally {
+        let RemoveScriptContent: string[] = [];
+        let RemoveAzContextPath = path.join(path.resolve(__dirname), 'RemoveAzContext.ps1');
+        let removeScripts = `${RemoveAzContextPath} -ErrorAction continue`
+        RemoveScriptContent.push(removeScripts);
+        const removeScriptFilePath = path.join(tempDirectory, uuidV4() + '.ps1');
+        try {
+            await fs.writeFile(
+                removeScriptFilePath,
+                '\ufeff' + RemoveScriptContent.join(os.EOL), // Prepend the Unicode BOM character.
+                { encoding: 'utf8' }, // Since UTF8 encoding is specified, node will
+                                            // encode the BOM into its UTF8 binary sequence.
+                function (err) {
+                    if (err) throw err;
+                });
+            const powershell = tl.tool(tl.which('pwsh') || tl.which('powershell') || tl.which('pwsh', true))
+                .arg('-NoLogo')
+                .arg('-NoProfile')
+                .arg('-NonInteractive')
+                .arg('-ExecutionPolicy')
+                .arg('Unrestricted')
+                .arg('-Command')
+                .arg(`. '${removeScriptFilePath.replace(/'/g, "''")}'`);
```

Comment:
I am sorry that I did not realized that before. But why do you need to generate a new temp script file that will always have the same content? Why don't you just use `RemoveAzContextPath` here instead of a path to the generated file?

Reply:
@onetocny Done as suggested.
Thank you!